internal/transport/controlbuf.go

```diff
@@ -505,7 +505,9 @@ func (l *loopyWriter) run() (err error) {
 			// 1. When the connection is closed by some other known issue.
 			// 2. User closed the connection.
 			// 3. A graceful close of connection.
-			infof("transport: loopyWriter.run returning. %v", err)
+			if logger.V(logLevel) {
```

Comment:
Can we keep the helper `infof()` and check `V()` there?

Reply:
I removed them because @dfawley suggested they weren't needed. This also avoids the formatting overhead when the verbosity condition fails.